### ISSUE TITLE
Several buttons are missing AccessibilityLabel strings for VoiceOver users

### ISSUE THREAD

**Issue by isaackeyet** [wordpress-mobile/WordPress-iOS#72]
Examples are the buttons at the bottom of a comment view to moderate, mark as spam etc. Some of these buttons have a tendency to lose these AccessibilityLabels even when they are fixed in interim builds.
They should be added in code by using the SetAccessibilityLabel method, similar to the side bar toggle in the main window, and tested for so this does not continually regress.
For both iPhone & iPad obviously...

---

OK, first tests were successful. Feedback on the attached patch is welcome, esp about the two buttons near the top left that are very close together. The left one currently carries the label "Toggle Sidebar", the other the label "Comments", but I do not know if this is accurate.
Also, are you aware of any buttons I might have missed? I have not found any yet, but the code base is not small. :)

http://ios.trac.wordpress.org/attachment/ticket/1601/AccessibilityLabels.txt

---

One question about http://ios.trac.wordpress.org/attachment/ticket/1601/AccessibilityLabels.txt#L206
Would you comment on why the AccessibilityLabel for the toggle button should be removed? I'm not sure I understand.
Otherwise I think the patch is good. 

---

Updated and re-based patch with request for comment addressed

http://ios.trac.wordpress.org/attachment/ticket/1601/AccessibilityLabels.txt

---

Replying to aerych:
One question about http://ios.trac.wordpress.org/attachment/ticket/1601/AccessibilityLabels.txt#L206
Would you comment on why the AccessibilityLabel for the toggle button should be removed? I'm not sure I understand.
I put a comment in the file stating why the label should not be set at the location it was previously. Basically, it would cover up the real button, which is one of two buttons which get added to this -- let me call it -- container button. One is the sidebar toggle, the other one is the notifications button. Funny construct there, but this is the best way I found it to work with VoiceOver.
I attached a new patch with some additional labels for the reader post detail view, for the share, like, comment, and re-blog buttons.

---

Originally reported by MarcoZ at http://ios.trac.wordpress.org/ticket/1601 (see thread there for reference)
